presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -2099,10 +2099,7 @@ private List<FunctionCall> analyzeWindowFunctions(QuerySpecification node, List<
                 }
 
                 Window window = windowFunction.getWindow().get();
-                if (window.getOrderBy().filter(
-                                orderBy -> orderBy.getSortItems().stream().anyMatch(
-                                        item -> item.getSortKey() instanceof Literal))
-                        .isPresent()) {
+                if (window.getOrderBy().filter(orderBy -> orderBy.getSortItems().stream().anyMatch(item -> item.getSortKey() instanceof Literal)).isPresent()) {
```

Comment:
unrelated change?

Reply:
This line had checkstyle violations.

Reply:
oh probably just belongs in the first commit.  looks like when you added the functionAndtypeManager in this class you reformatted this line too.